dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/preheat/TrackerPreheat.java

```diff
@@ -127,6 +130,11 @@
      */
     private Map<TrackerIdScheme, Map<String, Relationship>> relationships = new EnumMap<>( TrackerIdScheme.class );
 
+    /**
+     * Identifier map
+     */
+    private TrackerIdentifierParams identifiers = new TrackerIdentifierParams();
+
     public TrackerPreheat()
```

Comment:
A couple of things I noticed on this class:
`uniqueTrackedEntityAttributes` mutators are never accessed
`mandatoryProgramAttributes` mutators are never accessed

Reply:
This is a good catch, this is not implemented yet.
We need to discuss this next meeting.
Added to the list.